app/services/post_alerter.rb

```diff
@@ -319,32 +319,44 @@ def create_notification(user, type, post, opts = {})
       ).exists?
     end
 
-    # Don't notify the same user about the same notification on the same post
-    existing_notification = user.notifications
+    existing_notifications = user.notifications
       .order("notifications.id DESC")
-      .find_by(
+      .where(
         topic_id: post.topic_id,
-        post_number: post.post_number,
-        notification_type: type
-      )
+        post_number: post.post_number
+      ).limit(10)
+
+    existing_notification_of_same_type = existing_notifications.find { |n| n.notification_type == type }
+
+    # existing_notification = user.notifications
```

Comment:
Why all these comments?

Reply:
Oh my, I didn't clean up properly when I pushed this last Friday.